html/changelogs/Kurfurst.yml

```diff
@@ -0,0 +1,18 @@
+author: Kurfurst
```

Comment:
I think it's fair you take credit for this one after all the work you've done reviving it

Reply:
He added a changelog in the PR, so his name'll be on / near it as well.